.github/ISSUE_TEMPLATE/1-bug.md

```diff
@@ -32,6 +32,12 @@ Tell us how to reproduce this issue.  Please provide a working and simplified ex
 
 🎉 BONUS POINTS for creating a [minimal reproduction](https://stackoverflow.com/help/mcve) and uploading it to GitHub.  This will get you the fastest support.  🎉
 
+<!--
+Consider using a codesandbox to make the issue easy to replicate, here are two starters:
+https://codesandbox.io/s/remark-debug-ikwvx
+https://codesandbox.io/s/react-markdown-debug-9n4eg
```

Comment:
```suggestion
remark only (for markdown to markdown): https://codesandbox.io/s/remark-debug-ikwvx
remark and rehype (for markdown to html): xxx
react-markdown: https://codesandbox.io/s/react-markdown-debug-9n4eg
```

Reply:
well that was fast: https://github.com/remarkjs/remark/discussions/559

Reply:
added in 8ab25ed